pkg/controller/operators/catalog/operator_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"log"
```

Comment:
this is likely a mistake from `goimports` or a similar tool - we use logrus everywhere, or `t.Log` for tests.